cmd/api/src/daemons/datapipe/ingest.go

```diff
@@ -44,21 +45,44 @@ func ReadFileForIngest(batch graph.Batch, reader io.ReadSeeker) error {
 	}
 }
 
-func IngestBasicData(batch graph.Batch, converted ConvertedData) {
-	IngestNodes(batch, ad.Entity, converted.NodeProps)
-	IngestRelationships(batch, ad.Entity, converted.RelProps)
+func IngestBasicData(batch graph.Batch, converted ConvertedData) error {
+	errs := []error{}
+
+	if err := IngestNodes(batch, ad.Entity, converted.NodeProps); err != nil {
+		errs = append(errs, err)
+	} else if err = IngestRelationships(batch, ad.Entity, converted.RelProps); err != nil {
+		errs = append(errs, err)
+	}
```

Comment:
Previously these would continue on to the next step if there was an error. Now they will not due to the else-if-tree. Is this intentional? If not, these would need to be split into individual if statements.